### ISSUE TITLE
Help with UgCS

### ISSUE THREAD

**Issue by belardo98** [ugcs/mavlink-java#1]
Hi, I was wondering If I could get into contact with you? I'm looking to make UgCS accept a value that will allow me to tilt the camera up. If you know if there is any way to do this please let me know

**Comment by ayankelevich-ugcs** [comment]
Do you want to control camera tilt along the waypoint flight (mission) or using joystick?

**Comment by belardo98** [comment]
Yeah I would like to have the camera set to automatically tilt when I set
it on a way point.

Cheers

**Comment by ayankelevich-ugcs** [comment]
We mostly support .Net SDK . Please take a look https://github.com/ugcs/ugcs-dotnet-sdk/tree/master/OtherExamples/RouteUploading . I suppose we can add example how to construct route with waypoints and specify tilt for camera

**Comment by belardo98** [comment]
Sorry I am a bit new to all this, and my coding experience is very limited.
But through using this method would I still maintain all of the normal
functionality of UgCS? I only need the camera to look up twice during my
route if that makes things any easier?

I was wondering how this would interact with the UgCS android app, would
there be any sticking points where the app perhaps would not accept a value
outside of the normal range? Could we try to upload the route completely
independently from the app to subvert this?

Thank you for assisting me so far it's been really helpful
